coffee-dto/coffee-dto-base/src/main/java/hu/icellmobilsoft/coffee/dto/exception/BaseExceptionWrapper.java

```diff
@@ -70,7 +77,9 @@
  * @author ferenc.lutischan
  * @since 1.0.0
  */
-public interface BaseExceptionWrapper<E extends BaseException> {
+@Deprecated(since = "2.12.0")
```

Comment:
Deprecation should be documented in the javadoc with the new expected alternative.